injectLatestProps.js

```diff
@@ -1,5 +1,28 @@
-const { exec } = require("child_process");
-exec(
+const { execSync } = require("child_process");
+
+execSync("rm -rf ./react-native-elements/", (err, stdout, stderr) => {
```

Comment:
rm -rf is not recommended. Use rimraf

Reply:
sure.

Reply:
@pranshuchittora , added `rimraf` please review.